tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -24,6 +24,8 @@ namespace {
 
 // In case of failure, every call will be retried kMaxAttempts-1 times.
 constexpr int kMaxAttempts = 4;
+// Maximum backoff time in seconds
+constexpr int maximumBackoffSeconds = 32;
```

Comment:
Could you please rename this to kMaximumBackoffSeconds in order to be consistent with the naming conventions for constants?